lib/ai_client.rb

```diff
@@ -0,0 +1,72 @@
+require "openai"
+
+class AiClient
+  DIFF_LIMIT = 10000
+
+  attr_reader :access_token, :client
+
+  def initialize(access_token = nil)
+    @access_token = access_token || Envs::OpenaiApiToken.new.fetch!
+    @client = OpenAI::Client.new(access_token: @access_token)
+  end
```

Comment:
allow passing `access_token` directly for user_input validation in `def validate_user_input!(token)` in `Envs::OpenaiApiToken`